ExtLibs/Xamarin/Xamarin/Resources/mavcmd.xml

```diff
@@ -210,7 +210,16 @@
       <Y></Y>
       <Z></Z>
     </DO_SET_CAM_TRIGG_DIST>
-    <DO_SET_RELAY>
+    <DO_SET_HOME>
+      <P1>Current(1)/Spec(0)</P1>
+      <P2></P2>
+      <P3></P3>
+      <P4></P4>
+      <X>Lat</X>
+      <Y>Long</Y>
+      <Z>Alt</Z>
+    </DO_SET_HOME>
+     <DO_SET_RELAY>
```

Comment:
nitpick: there's one extra character at the front I think

Reply:
will fix